components/base/kinematicbase/ptgKinematics.go

```diff
@@ -223,6 +223,10 @@ func (ptgk *ptgBaseKinematics) ErrorState(ctx context.Context, plan [][]referenc
 		return nil, err
 	}
 	nominalPose = spatialmath.Compose(runningPose, currPose)
+	ptgk.logger.Debugf("curr position %#v", spatialmath.PoseToProtobuf(actualPIF.Pose()))
```

Comment:
I'm not sure we want these to be actually merged into main

Reply:
removed